docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-components-con.adoc

```diff
@@ -0,0 +1,58 @@
+[id='building-custom-dashboard-widgets-components-con']
+= Components
+
+A page contains different types of components. You can choose from the following components types, for using them in a page:
+
+* *Core* components: Used for displaying data. They can be a new, blank HTML page or an existing page. There are two types of core components.
+
+.Core components sub-types
+[cols="1,1", options="header"]
+|===
+| Core component sub-types
+| Description
+
+|*HTML*
+|This component opens a HTML editor widget. This can be used to create HTML pages using text, images, tables, links, colors, and so on. You can customize the page as per your requirements.
+|*Page*
+|This component allows you to add previously created pages to a new dashboard. You can use this component to nest custom pages you have already created in your dashboard.
+
+|===
+
+[NOTE]
+=======
+Core components are not mandatory.
+=======
+
+* *Navigational* components: Used for navigating between pages. There are six types of navigational components.
+
+.Navigational components sub-types
+[cols="1,1", options="header"]
+|===
+| Navigational component sub-types
+| Description
+
+|*Target Div*
+|This component is used by other components (Tab list, Menu bar, Tree navigator) to display their entries and keep track of the last item clicked.
+|*Menu Bar*
+|This component displays the entries of a navigation tree in the form of a menu bar. There is no limit to the number of levels supported by {PRODUCT}.
+|*Tile Navigator*
+|This component displays the navigation group in the form of tiles. *Workbench* is the default navigation group, when you create a new tile navigator. Groups are displayed as folders but for single entries, its content is displayed instead when selected.
+|*Tree*
+|This component displays the entries in a vertical tree structure format.
+|*Carousel*
+|This component displays or cycles through the selected perspectives like a carousel or a slide show.
+|*Tab List*
+|This component displays selected menu perspectives as tabs at the top of the widget.
```

Comment:
"perspectives" => pages
"the top of the widget" => the top of the component

Reply:
changed as suggested.